### PR TITLE
fix: import client-go authProviders

### LINKED ISSUE #2: build error: missing requirements
Just did a fresh clone `Tue Sep 21 12:42:46 2021 +0530` and getting the following build error when running the make command.

```
go build -o bin/kubedd .
kubedd/kubedd.go:23:2: package github.com/devtron-labs/silver-surfer/pkg imports github.com/mgutz/ansi from implicitly required module; to add missing requirements, run:
	go get github.com/mgutz/ansi@v0.0.0-20200706080929-d51e80ef957d
make: *** [build] Error 1
```

I ran the following command to resolve the issue.

```
go mod tidy
go mod vendor
```

After that make was able to run and the binary executes OK.

### LINKED ISSUE #3: panic: runtime error: invalid memory address or nil pointer dereference
When running kubedd with no arguments I get a panic:

```
goroutine 1 [running]:
github.com/devtron-labs/silver-surfer/pkg.(*Cluster).ServerVersion(0xc0001b3940)
	/go/src/silver-surfer/pkg/Cluster.go:60 +0x18
github.com/devtron-labs/silver-surfer/kubedd.ValidateCluster(0x0, 0xc00046e8c0)
	/go/src/silver-surfer/kubedd/kubedd.go:95 +0x145
main.processCluster()
	/go/src/silver-surfer/main.go:168 +0x251
main.glob..func1(0x2b2a240, {0x2b6f4c8, 0x0, 0x0})
	/go/src/silver-surfer/main.go:108 +0x12c
github.com/spf13/cobra.(*Command).execute(0x2b2a240, {0xc000198240, 0x0, 0x0})
	/go/src/silver-surfer/vendor/github.com/spf13/cobra/command.go:766 +0x5f8
github.com/spf13/cobra.(*Command).ExecuteC(0x2b2a240)
	/go/src/silver-surfer/vendor/github.com/spf13/cobra/command.go:852 +0x2dc
github.com/spf13/cobra.(*Command).Execute(...)
	/go/src/silver-surfer/vendor/github.com/spf13/cobra/command.go:800
main.Execute()
	/go/src/silver-surfer/main.go:258 +0x25
main.main()
	/go/src/silver-surfer/main.go:286 +0x17
```

### PR DESCRIPTION
**Pull request by Meroje**
This fixes OP's issue on #3, I'll open a separate issue for the path resolution

Fixes #3
Fixes #2 by importing the missing ansi lib

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -7,7 +7,7 @@ require (
 	github.com/getkin/kin-openapi v0.67.0
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/inconshreveable/mousetrap v1.0.0 // indirect
-	github.com/olekukonko/tablewriter v0.0.5 // indirect
+	github.com/mgutz/ansi v0.0.0-20200706080929-d51e80ef957d
```
Comment (pghildiyal) [inline go.mod:10]: Are these changes required in go.mod?

Comment (Meroje) [reply go.mod:10]: yes
```
make
go build -o bin/kubedd .
kubedd/kubedd.go:23:2: package github.com/devtron-labs/silver-surfer/pkg imports github.com/mgutz/ansi from implicitly required module; to add missing requirements, run:
	go get github.com/mgutz/ansi@v0.0.0-20200706080929-d51e80ef957d
```

**Review by pghildiyal — APPROVED** [review]
